dev/src/dev.clj

```diff
@@ -297,6 +298,27 @@
                                  (qp.compile/compile built-query))]
     (into [query] params)))
 
+(methodical/defmethod t2.hydrate/hydrate-with-strategy :around ::t2.hydrate/multimethod-simple
+  "Throws an error if do simple hydrations that make DB call on a sequence."
+  [model strategy k instances]
+  (if (or config/is-prod?
```

Comment:
Do we still need this check if we register this method this in dev?

Reply:
Better safe than sorry :P

Reply:
if you start your repl with `:dev` alias, then you won't miss anything.